docs/get-started/getting-started-alluxio.md

```diff
@@ -365,3 +365,49 @@ This section will give some links about how to configure, tune Alluxio and some
 - [Running Spark on Alluxio](https://docs.alluxio.io/os/user/stable/en/compute/Spark.html)
 - [Performance Tuning](https://docs.alluxio.io/os/user/stable/en/operation/Performance-Tuning.html )
 - [Alluxio troubleshooting](https://docs.alluxio.io/os/user/stable/en/operation/Troubleshooting.html)
+
+## Alluxio reliability
+The properties mentioned in this section can be found in [Alluxio configuration](https://docs.alluxio.io/os/user/stable/en/reference/Properties-List.html)
+
+### Avoid long time response if masters or workers hang
+The default value of `alluxio.user.rpc.retry.max.duration` is 2 minutes.  
+If master is done or hang, the `alluxio.user.rpc.retry.max.duration` will cause at least 2 minutes waiting, it's too long.
+The default value of `alluxio.user.block.read.retry.max.duration` is 5 minutes.
+If worker is done or hang, the `alluxio.user.block.read.retry.max.duration` will cause 5 minutes waiting, it's too long.
```

Comment:
```suggestion
If the worker is not responding, possibly due to it crashing or GC pause , `alluxio.user.block.read.retry.max.duration` will cause the client to retry for 5 minutes.  This is a very long time and can cause delays in the running job, so we suggest lowering this value to 10 seconds.
```

Reply:
Actually this one worries me a bit, the definition is "This duration controls for how long Alluxio clients should try reading a single block. ".  So this implies to me it includes the time you are streaming the read.  So if you just have a very large block and are reading from say s3, this time seems low to me.  I would rather see this set a bit higher then 10 seconds. maybe 60 seconds.

Reply:
as the comment says the chunk size is configurable.  In my opinion there is a trade off here of failing quickly vs once in a while getting something slow downloading.  Failing to fast on something like this in my opinion could be bad, especially if it tries again and fails again and again. There is a reason they have the defaults higher and since this includes the time to actually get a block, I would rather see this higher  60 seconds. We can tune it more later if we see issues but I don't want to arbitrarily cause issues because its to small.

Reply:
Yes， this explanation is very reasonable, changed to 1 minute.